### DISCUSSION TITLE
Tailwind classes do not get generated

### DISCUSSION THREAD

**Discussion by toughyear** [tailwindlabs/tailwindcss#8894] [Help]
I recently upgraded our react app from CRA 4 to CRA 5. 

I installed Tailwind following this doc - https://tailwindcss.com/docs/guides/create-react-app.

When I run the server, the classes have no effects on the component. I can see the classname on the component, but I don't see that class definition in chrome styles inspector.
![image](https://user-images.githubusercontent.com/38691925/179725628-2081d8b4-a669-4697-848f-36a452629fb5.png)

The component -
![image](https://user-images.githubusercontent.com/38691925/179725685-79e10513-2b8e-4d79-a732-b49a16a5804b.png)

This is what I see in the source - 
![image](https://user-images.githubusercontent.com/38691925/179725832-fb3b2a6b-7420-4023-b235-770a21425b0e.png)

My assumption is that no classes are being generated as I can't find any in the source.

Help!

**Comment by arisac** [comment]
Should be mis-configs in your project. Please follow every steps of the instructions.

Make sure your `app.js / index.js` (may in other names dep on your project) has `import 'styles/index.css'`

If you still need help, better create a demo repo that can reproduce your problem for other people to help on your issue.
